### PR TITLE
doc: dev improve the s3tests doc to reflect current scripts

### PR DESCRIPTION
**Pull request by theanalyst**
minor improvements to the s3tests runner
- make the workunit s3tests task to default to python2
- add the current cmake build path to the workunit
- drop the old info on the s3tests suite since the workunit does all of that

### CODE REVIEW

File: doc/dev/dev_cluster_deployement.rst [hunk 1]
```diff
@@ -73,7 +69,7 @@ Options
 
     Launch the osd/mds/mon/all the ceph binaries using valgrind with the specified tool and arguments.
 
-.. option:: --{mon,osd,mds}_num
+.. option:: --{mon,osd,mds,rgw}_num
```
Comment (liewegas) [inline doc/dev/dev_cluster_deployement.rst]: I'd prefer we got rid of these and consolidate support on OSD= MDS= MON= etc.

File: doc/dev/index.rst [hunk 2]
```diff
@@ -1464,51 +1450,13 @@ end you should see a message like::
...
-
-    S3TEST_CONF=/path/to/s3.conf ./virtualenv/bin/nosetests -a '!fails_on_rgw' -v
-
-Note: the following test is expected to error - this is a problem in the test
-setup (WIP), not an actual test failure::
-
-    ERROR: s3tests.functional.test_s3.test_bucket_acl_grant_email
+To run the s3tests suite do the following::
 
+   $ ../qa/workunits/rgw/run-s3tests.sh
```
Comment (liewegas) [inline doc/dev/index.rst]: pretty sure you need to prefix that with PATH=bin:$PATH

**Comment by liewegas** [comment]
otherwise lgtm!

File: doc/dev/dev_cluster_deployement.rst (hunk 1 above)
Comment (theanalyst) [inline doc/dev/dev_cluster_deployement.rst]: Sounds good, also would work across all versions

**Comment by theanalyst** [comment]
@liewegas updated

**Comment by theanalyst** [comment]
@liewegas ping

**Pull request merged**